.cloud-build/execute_changed_notebooks_helper.py

```diff
@@ -80,6 +86,45 @@ def output_uri_web(self) -> Optional[str]:
         else:
             return None
 
+def load_results(results_bucket: str,
+                 results_file: str) -> Dict[str, Any]:
+    '''
+    Load accumulated notebook test results
+    '''
+
+    print("Loading existing accumulative results ...")
+    accumulative_results = {}
+    try:
+
+        storage_client = storage.Client() 
+        bucket = storage_client.get_bucket(results_bucket)
+        blob = bucket.blob(results_file)
+        accumulative_results = json.loads(blob.download_as_string(client=None))
```

Comment:
```
results_as_json = json.loads(blob.download_as_string(client=None))
results = [NotebookExecutionResult(**result) for result in results]
```

Reply:
This uses dictionary unpacking to instantiate the result. See https://python-reference.readthedocs.io/en/latest/docs/operators/dict_unpack.html